scripts/detectNotUsingGitPush1by1.fsx

```diff
@@ -794,11 +815,47 @@ let GitHubApiCall(url: string) =
     client.DefaultRequestHeaders.Add("User-Agent", userAgent)
     client.DefaultRequestHeaders.Add("X-GitHub-Api-Version", xGitHubApiVersion)
 
-    (client.GetStringAsync url).Result
+    if not(String.IsNullOrEmpty accessToken) then
+        client.DefaultRequestHeaders.Add(
+            "Authorization",
+            $"token {accessToken}"
+        )
+
+    let result =
+        try
+            (client.GetStringAsync url).Result
+        with
+        | :? AggregateException as aggregateException ->
+            match aggregateException.InnerException with
+            | :? HttpRequestException as httpRequestException ->
+                match httpRequestException.StatusCode with
+                | statusCode ->
+                    if statusCode.HasValue
+                       && statusCode.Value = HttpStatusCode.NotFound then
+                        let errorMsg =
+                            """
+Please add the \"ACCESS_TOKEN\" environment variable to the GitHubCI file:
```

Comment:
@realmarv this text has to start after the 3 double-quote chars, not after an EOL

Reply:
Done